src/libraries/icubmod/embObjMotionControl/CMakeLists.txt

```diff
@@ -6,7 +6,7 @@ yarp_prepare_plugin(embObjMotionControl
     CATEGORY device
     TYPE yarp::dev::embObjMotionControl
     INCLUDE embObjMotionControl.h
-    EXTRA_CONFIG WRAPPER=controlboardwrapper2)
+    EXTRA_CONFIG WRAPPER=controlBoard_nws_yarp)
```

Comment:
Apparently, this line serves to generate the following file:
```c++
//...
#ifdef YARP_STATIC_PLUGIN
YARP_PLUGIN_EXPORT void add_owned_embObjMotionControl(const char *owner) {
    yarp::dev::DriverCreator *factory =
        new yarp::dev::DriverCreatorOf<yarp::dev::embObjMotionControl>("embObjMotionControl",
                "controlBoard_nws_yarp",
                "yarp::dev::embObjMotionControl");
    yarp::dev::Drivers::factory().add(factory); // hand factory over to YARP
}
#endif

YARP_DEFINE_SHARED_SUBCLASS(embObjMotionControl, yarp::dev::embObjMotionControl, yarp::dev::DeviceDriver)
```

In turn, the YARP class `yarp::dev::DriverCreatorOf<>` receives this information to link `controlBoard_nws_yarp` as a possible wrapper.

It seems there ain't particular problems here.
@Nicogene do you know other impacts of the use of our `EXTRA_CONFIG_WRAPPER` CMake key?

cc @marcoaccame